tests/test_parse_deep.py

```diff
@@ -14,6 +17,8 @@ class TestParseDeep(unittest.TestCase):
     def test_parse_article_02935_simple(self):
         "some simple comparisons and count list items"
         article_object, error_count = parse.build_article_from_xml(XLS_PATH + 'elife-02935-v2.xml')
+        # test pretty method for test coverage
+        self.assertIsNotNone(article_object.pretty())
```

Comment:
could be stricter and test the return value is a string?

Reply:
Upon checking it actually returns an `OrderedDict`:
```
def pretty(self):
    "sort values and format output for viewing and comparing in test scenarios"
    pretty_obj = OrderedDict()
    for key, value in sorted(iteritems(self.__dict__)):
        if value is None:
            pretty_obj[key] = None
        elif is_str_or_unicode(value):
            pretty_obj[key] = self.__dict__.get(key)
        elif isinstance(value, list):
            pretty_obj[key] = []
        elif isinstance(value, dict):
            pretty_obj[key] = {}
        else:
            pretty_obj[key] = unicode_value(value)
    return pretty_obj
```
I avoided changing the actual test assertions, I could change this one obviously its only a small change but even just in that test file there is about 30+ other cases that could check return type but just in fact make sure the return type is not `None`.

I will leave for now, but agree could be picked up during a refactor.

Reply:
As above, the test is really only there for test coverage. If no one can think of a use case for ``pretty()`` then we can delete the function entirely.